docs/security/additional-security-features/introduction-to-additional-security-features.md

```diff
@@ -1,29 +1,23 @@
 ---
-id: introduction-to-cloud-infrastructure-security
-title: Introduction to Cloud Infrastructure Security
-sidebar_label: Introduction to Cloud Infrastructure Security
-description: Learn basic concepts about Cloud Infrastructure Security. 
+id: introduction-to-additional-security-features
```

Comment:
"additional-security-features" is already in the URL

```suggestion
id: introduction
```
<img width="627" alt="Screenshot 2024-05-31 at 3 32 57 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/5085268d-601b-48db-9c99-1899dc8e5731">